tests/test_admin_views.py

```diff
@@ -42,7 +42,7 @@ def test_logout(admin_client):
     """
     url = reverse("admin:logout")
 
-    response = admin_client.get(url)
+    response = admin_client.post(url)
```

Comment:
Interesting this wasn't already done, as I thought we fixed the logout issue already

Reply:
The logout via GET was deprecated in Django 4.1 hence why it wasn't earlier
https://docs.djangoproject.com/en/4.2/releases/4.1/#log-out-via-get

Reply:
Yeah, I think it might have been missed in https://github.com/farridav/django-jazzmin/pull/544